doc/userDocs/usersGuide.md

```diff
@@ -1701,3 +1702,31 @@ configuration file you should:
 
 }
 ```
+
+## Appendix<a name="appendices"></a>
+
+### Storage and Network Resource Demands
+
+This section gives the typical storage and network demands created by the Weathervane 
+application for each configuration.  These demands are per WvUser, meaning the the 
+demands for a given run can be estimated by multiplying the expected number of WvUsers 
+by the values given in the tables. There will be some run-to-run variation due to randomness 
+in the workload.  Storage demands tend to vary more widely than the network demands.
```

Comment:
Where are these metrics measured at?   At the sut host level?  Does it include driver?